src/plugins/interactive_setup/server/elasticsearch_service.ts

```diff
@@ -147,9 +168,6 @@ export class ElasticsearchService {
     { apiKey, hosts, caFingerprint }: EnrollParameters
   ): Promise<EnrollResult> {
     const scopeableRequest: ScopeableRequest = { headers: { authorization: `ApiKey ${apiKey}` } };
-    const elasticsearchConfig: Partial<ElasticsearchClientConfig> = {
```

Comment:
__issue:__ any reason you removed this config from `enrollClient`? I believe we won't be able to connect to ES without `ssl: { verificationMode: 'none' },`

Reply:
Yeah, just checked - I cannot enroll without this. Did it work for you?

Reply:
Bad merge, thanks for spotting.